bin/plotting/pycbc_plot_dq_flag_likelihood

```diff
@@ -37,24 +40,25 @@ color = pycbc.results.ifo_color(ifo)
 
 fig = pyplot.figure(0)
 ax = fig.add_subplot(111)
-ax.bar(x, height=logrates, label=f'{ifo} DQ Flag Trigger Rates',
+ax2 = ax.twinx()
+
+ax2.bar(x, height=logrates, label=f'{ifo} DQ Flag Trigger Rates',
```

Comment:
I think having just the ifo name as label would be fine, the y-axes already explain what the values mean, which in fact is two different things.  (Actually, how about allowing the plot to show multiple ifos at once?  Just an idea ..)

Reply:
```suggestion
ax2.bar(x, height=logrates, label=f'{ifo} Triggers',
```